Xamarin.Forms.GoogleMaps/Xamarin.Forms.GoogleMaps.Android/MapRenderer.cs

```diff
@@ -51,7 +51,19 @@ public MapRenderer() : base()
 
         protected GoogleMap NativeMap { get; private set; }
 
-        protected Map Map => Element;
+        private Map map;
+        protected Map Map
+        {
+            get { return map ?? Element; }
+            set { map = value; }
+        }
+
+        MapView mapView;
+        public MapView MapView
+        {
+            get { return mapView ?? (MapView)Control; }
+            set { mapView = value; }
```

Comment:
Improve accessibility.
``set { mapView = value; }`` to ``private set { mapView = value; }``